### DISCUSSION TITLE
Any way to get rid of number of open positions under Career Portal?

### DISCUSSION THREAD

**Discussion by sanmah01** [bullhorn/career-portal#404] [Ideas]
In the Bullhorn career portal that is embedded onto our website, there is the number of open positions besides the title "Open Positions."

![image](https://user-images.githubusercontent.com/87321394/126180944-d2fa519f-54bb-4f59-8af0-c72eebcd0a93.png)

We are okay with keeping the amount of positions in "Filter by Category" besides each category. However, we were hoping to eliminate the "(126)" next to "Open Positions." 

Is there anything specific I can do to get rid of it? Thank you!

**Comment by charlesabarnes** [comment]
Hello, there is no configuration option, but can be changed via code  Below I have linked the portion of code responsible.
https://github.com/bullhorn/career-portal/blob/master/src/app/job-list/job-list.component.html#L7

**Reply by charlesabarnes** [reply]
I am also converting this to the `Ideas`  category to triage as a feature request
